### DISCUSSION TITLE
what is the estimated time for zim-requests to be created?

### DISCUSSION THREAD

**Discussion by hamoudak** [openzim/zim-requests#1118] [Q&A] [answered]
I've requested two zims; since 1st May 2024. https://github.com/openzim/zim-requests/issues/986
the second request was on May 29 : https://github.com/openzim/zim-requests/issues/1013
I don't know the estimated timing of these stuff or the rules set for it.

* another question: If there's  a net connection interruptions happen when I create a zim from https://zimit.kiwix.org/; would it affect somehow on the created zim file? it works fine though. I was getting a message : unable to retrieve (something) its probably CORS issue. I have used to ignore it and it continues processing the zim fine. sometimes I get the same message with no issues on net connection at the beginning of processing the request.

**Comment by Popolechien** [comment, marked as answer]
There is no set rule as to how zim requests are handled, other than best-effort.

In your specific case, however, we've been rolling out and testing zimit 2.0 for the better part of Q2, so there was no new zim possibly coming out. We now need to go through the backlog ([233 requests ](https://github.com/openzim/zim-requests/issues?q=is%3Aopen+is%3Aissue+-label%3ABug+-label%3ADuplicate+-label%3AEnhancement+-label%3AInvalid+-label%3Aip_whitelisting+-label%3ARemove+-label%3A%22Scraper+Needed%22+-label%3ASponsored+-label%3AUpstream) as of 23.07), so we'll eventually make it. But, we can't set a timeframe for when that'll happen as more often than not it is not a matter of simply putting a URL in a box.

**Reply by hamoudak** [reply]
thank you for your response; I appreciate it.  I thought so too that there's no rule but to be sure, so I should wait for my zim-requests.  the second question, do I have to worry about the created zim file with interrupted connection although it works fine? for the  a bove message, is it normal to see it  when my net connection has no issues ?

**Reply by Popolechien** [reply]
As the issue creator you should normally get a notification when the request is being handled so yes, you can simply sit back and wait.

As for your other question, I suspect it really depends on the type of interruption, and you seem to confirm that zimit will just move on and continue crawling. But to be sure I'd open an issue at https://github.com/openzim/zimit/issues. Do add any log you have that mentions your problem so everybody is on the same page.
